### ISSUE TITLE
IE and Server.js not being a vaild filename

### ISSUE THREAD

**Issue by rodneyrehm** [theintern/intern#631]
For some reason tests in IE started failing on BrowserStack. Any Idea what the problem could be that Intern is trying to load `Server.js` and failing at it?

```
‣ Created session IE11 on WIN (<session>)
Suite IE11 on WIN - unit tests FAILED
UnknownError: [POST https://(redacted)@hub.browserstack.com:443/wd/hub/session/<session>/url / {"url":"http://localhost:9000/__intern/client.html?config=test%2Fbrowserstack&basePath=%2F&initialBaseUrl=%2F&reporters=%7B%22id%22%3A%22WebDriver%22%7D&rootSuiteName=IE11%20on%20WIN%20-%20unit%20tests&sessionId=<session>"}] Session not started or terminated
```

![loading Server.js](https://s3.amazonaws.com/testautomation/1552aed47c50b644539c52d09dee841787949c4b/screenshot-23a7cd2b21.jpeg)

**Comment by jason0x43** [comment]
This is related to https://github.com/theintern/leadfoot/issues/78. The problem is that a new Leadfoot feature test was added for file upload capability, and it's mysteriously breaking for some users.

**Comment by jason0x43** [comment]
This is fixed in release 3.2.1.
